user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component.md

```diff
@@ -209,15 +209,18 @@ You can do this in several different ways, for example:
 
 - Specify a **text string in the dashboard or app URL**:
 
-  1. Hover over the component, click the filter icon, and then add a filter feed from the *Feeds > URL > Strings* section of the data pane.
+  1. Hover over the component, click the filter icon, and then add a filter feed from the *Feeds > URL > Texts* section of the data pane (prior to DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12: *Feeds > URL > Strings*).
 
   1. Pass a string data object within the URL, as explained in [Specifying data input in a dashboard or app URL](xref:Specifying_data_input_in_a_dashboard_URL).
 
      This URL will automatically display a filtered version of the table when the dashboard or app is opened.
 
-     In the following example, the string "test" is sent to the component with component ID 1:
+     In the following example, the text string "test" is sent to the component with component ID 1:
 
-     `https://<dma>/<app-id>?data={"components": [{"cid":1, "select":{"strings": ["test"]}}]`
+     `https://<dma>/<app-id>?data={"components": [{"cid":1, "select":{"texts": ["test"]}}]`
```

Comment:
This has not changed, should remain "strings":

Reply:
```suggestion
     `https://<dma>/<app-id>?data={"components": [{"cid":1, "select":{"strings": ["test"]}}]`
```